fondat/aws/bedrock/resources/sessions.py

```diff
@@ -0,0 +1,289 @@
+            params["sessionMetadata"] = sessionMetadata
+        async with runtime_client(self.config_runtime) as client:
+            return await client.update_session(**params)
+
+    @property
+    def invocations(self) -> "InvocationsResource":
+        """Get invocations resource."""
+        return InvocationsResource(
+            self._agent_id, config_runtime=self.config_runtime, policies=self.policies
+        )
+
+
+@resource
+class InvocationsResource:
+    """Resource for managing session invocations."""
+
+    __slots__ = ("_agent_id", "config_runtime", "policies")
+
+    def __init__(
+        self,
+        agent_id: str,
+        *,
+        config_runtime: Config | None = None,
+        policies: Iterable[Policy] | None = None,
+    ):
+        self._agent_id = agent_id
+        self.config_runtime = config_runtime
+        self.policies = policies
+
+    @operation(method="post", policies=lambda self: self.policies)
+    async def create(
+        self,
+        sessionIdentifier: str,
+        invocationId: str,
+        *,
+        description: str | None = None,
+    ) -> Mapping[str, Any]:
+        """Create invocation.
+
+        Args:
+            sessionIdentifier: Session identifier
+            invocationId: Invocation identifier
+            description: Invocation description
+
+        Returns:
+            Invocation details
+        """
+        params = {
+            "sessionIdentifier": sessionIdentifier,
+            "invocationId": invocationId,
+        }
+        if description is not None:
+            params["description"] = description
+        async with runtime_client(self.config_runtime) as client:
+            return await client.create_invocation(**params)
+
+    @operation(method="post", policies=lambda self: self.policies)
+    async def put_step(
+        self,
+        sessionIdentifier: str,
+        invocationIdentifier: str,
+        payload: dict[str, Any],
+        invocationStepTime: datetime,
+        *,
+        invocationStepId: str | None = None,
+    ) -> Mapping[str, Any]:
+        """Add invocation step.
+
+        Args:
+            sessionIdentifier: Session identifier
+            invocationIdentifier: Invocation identifier
+            payload: Step payload
+            invocationStepTime: Step timestamp
+            invocationStepId: Step identifier
+
+        Returns:
+            Step details
+        """
+        params = {
+            "sessionIdentifier": sessionIdentifier,
+            "invocationIdentifier": invocationIdentifier,
+            "payload": payload,
+            "invocationStepTime": invocationStepTime,
+        }
+        if invocationStepId is not None:
+            params["invocationStepId"] = invocationStepId
+        async with runtime_client(self.config_runtime) as client:
+            return await client.put_invocation_step(**params)
+
+    @operation(method="get", policies=lambda self: self.policies)
+    async def get_step(
```

Comment:
Same here : should we be using a `InvocationResource` and just use `GET` for that resource ?